pytest_fixtures/component/provision_vmware.py

```diff
@@ -4,12 +4,23 @@
 from robottelo.config import settings
 
 
+@pytest.fixture
+def vmware_version(request):
+    versions = {
+        'vmware7': settings.vmware.vcenter7.hostname,
+        'vmware8': settings.vmware.vcenter8.hostname,
+    }
+    return versions[getattr(request, 'param', 'vmware7')]
```

Comment:
Even though it's not important here, keep always the latest :)
```suggestion
    return versions[getattr(request, 'param', 'vmware8')]
```